### PR TITLE
Removed Unnecessary GetCvarString

### PR DESCRIPTION
**Pull request by oylsister**
When plugin start the OnConVarChanged will automatically get called anyway.

### CODE REVIEW

File: addons/sourcemod/scripting/fire_customizer.sp [hunk 1]
```diff
@@ -146,4 +143,4 @@ public void loadInferno()
 		AddFileToDownloadsTable(infernoSystem);
 		PrecacheGeneric(infernoSystem, true);
 	}	
-}
```
Comment (Stewart-Anubis) [inline addons/sourcemod/scripting/fire_customizer.sp:149]: Keys cannot be removed from the block.

File: addons/sourcemod/scripting/fire_customizer.sp [hunk 2]
```diff
@@ -67,16 +67,13 @@ public void OnPluginStart()
...
-	g_cCvar_CustomFireInferno = CreateConVar("sm_fire_inferno_name", "inferno_fx_alt", "Defines the particle system file to be used for the inferno particles. ");
+	g_cCvar_CustomFireIgnite = CreateConVar("sm_fire_ignite_name", "burning_fx_sblue", "Defines the particle system file to be used for the ignite particles.");
+	g_cCvar_CustomFireInferno = CreateConVar("sm_fire_inferno_name", "inferno_fx_alt", "Defines the particle system file to be used for the inferno particles.");
 
 	// Hook cvar changes. The particles must be precached before they're active, so the safest bet is at round start
 
 	g_cCvar_CustomFireIgnite.AddChangeHook(OnConVarChanged);
 	g_cCvar_CustomFireInferno.AddChangeHook(OnConVarChanged);
-	
-	g_cCvar_CustomFireIgnite.GetString(g_sIgniteName, sizeof(g_sIgniteName));
```
Comment (Stewart-Anubis) [inline addons/sourcemod/scripting/fire_customizer.sp:78]: This part I agree, unnecessary.

File: addons/sourcemod/scripting/fire_customizer.sp [hunk 3]
```diff
@@ -67,16 +67,13 @@ public void OnPluginStart()
 { 
 	HookEvent("round_start", OnRoundStart);
 
-	g_cCvar_CustomFireIgnite = CreateConVar("sm_fire_ignite_name", "burning_fx_sblue", "Defines the particle system file to be used for the ignite particles. ");
```
Comment (Stewart-Anubis) [inline addons/sourcemod/scripting/fire_customizer.sp:70]: This part is the creation of the convar and cannot be missed.

**Review by Stewart-Anubis — COMMENTED** [review]
Thanks for the edits but there are some problems, please review.

File: addons/sourcemod/scripting/fire_customizer.sp (hunk 1 above)
Comment (oylsister) [reply addons/sourcemod/scripting/fire_customizer.sp:149]: if you check there is also a green line that got add, this is github spaghetti issues. make sure you check a green line.

File: addons/sourcemod/scripting/fire_customizer.sp (hunk 3 above)
Comment (oylsister) [reply addons/sourcemod/scripting/fire_customizer.sp:70]: Nah I didn't delete a whole part. I simply backspace one, if you compare to mine.

**Pull request merged**